scripts/sphinx_extensions/options_from_kconfig/options_from_kconfig.py

```diff
@@ -32,7 +32,7 @@ def _monkey_patch_kconfiglib(kconfiglib):
         inside the parsed Kconfig file itself will be available to be used by
         this extension.
         '''
-        kconfiglib._SOURCE_TOKENS = frozenset({})
+        kconfiglib._SOURCE_TOKENS = kconfiglib._REL_SOURCE_TOKENS
```

Comment:
I changed monkey patch to keep the rsource - this will allow application to keep a tree structure of configuration (like we do) and since Zephyr kconfig is sourced it will be ignored.